docs/application/native/guides/machine-learning/machine-learning-inference.md

```diff
@@ -298,6 +298,135 @@ Followings are the available elements:
     char pipeline[] = "videotestsrc is-live=true ! videoconvert ! tensor_converter ! output-selector name=outs outs.src_0 ! tensor_sink name=sink0 async=false outs.src_1 ! tensor_sink name=sink1 async=false"
     ```
 
+- **If**
+
+    The if element (tensor_if) controls its src-pad based on the values (other/tensor(s)) of its sink-pad.
+
+    For example, you may skip frames if there is no object detected with high confidence.
+
+    The input/output stream data type is either ```other/tensor``` or ```other/tensors```.
+
+    - **Properties**
+      - compared-value: Compared Value, operand 1 (from input tensor(s)).
+        * A_VALUE:  Decide based on a single scalar value
+        * TENSOR_AVERAGE_VALUE: Decide based on a average value of a specific tensor
+        * CUSTOM: Decide based on a user defined callback
```

Comment:
@gichan-jang
Can I get a link to the API reference, I could not find the same. I would like to know more about properties. 
So, 
compared-value is a property and A_VALUE is ???

Reply:
`tensor_if` is a newly added element. With the addition of the `tensor_if`, the newly added APIs are `ml_pipeline_tensor_if_custom_register` and `ml_pipeline_tensor_if_custom_unregister`.
`compared-value`  is not directly related to new APIs but a property of `tensor_if`. See the 359 th line for an example.
The newly added documentation contains descriptions of `tensor_if` itself. If this is unnecessary, it seems better to leave only an explanation of the API.
Please give me your opinion.